x-pack/plugins/fleet/server/services/utils/agentless.ts

```diff
@@ -22,3 +21,14 @@ export const isAgentlessServerlessEnabled = () => {
 export const isAgentlessEnabled = () => {
   return isAgentlessCloudEnabled() || isAgentlessServerlessEnabled();
 };
+
+const AGENTLESS_API_BASE_PATH = '/api/v1/ess';
```

Comment:
I was on the fence about this but now that I am familiar with the deployment schedule for Serverless this makes sense to me.